File: Plus  Minus.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
   int n;
   double neg(0), pos(0), zero(0);
   cin>>n;
   int arr[n];
   for(int i=0;i<n;++i)
   {
       cin>>arr[i];
       if(arr[i]>0)
        pos++;
       else if(arr[i]<0)
           neg++;
       else
          zero++;
   }
   cout<<fixed<<setprecision(6)<<pos/n<<"\n"<<neg/n<<"\n"<<zero/n;
}

File: Plus  Minus.py
n=int(input().rstrip())
array=list(map(int, input().rstrip().split()))
pos=0
neg=0
zero=0
for i in array:
    if i>0:
        pos+=1
    elif i<0:
        neg+=1
    else:
        zero+=1
print('%.6f'%(pos/n),"\n"'%.6f'%(neg/n),"\n" '%.6f'%(zero/n))
    

